vitest.config.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    include: ['test/**/*.test.js'],
+    setupFiles: ['./test/helpers/chai-plugins.js'],
```

Comment:
looks like this PR is mostly working again. only outstanding thing is refactoring the top-level integrations tests over to vitest and dropping this `chai-plugins.js` file.

Reply:
oh TIL — yeah works for me!